wordpress-mobile/WordPress-Android
# Time Selection Feature

Implemented time selection for blogging reminders

Fixes #15123 

<img width=320 src=https://user-images.githubusercontent.com/990349/128148139-33f88c86-1e27-456c-a5a2-6698b2a47850.png />


To test:

- Go to site settings
- Find **Blogging reminders** setting and click on the item below it
- It should launch select days bottom sheet 
- bottom sheet should now have Notification time item, click on time
- It should show time selection dialog
- Select a time, AM, PM and time should reflect the selection
- De-select all days, time list item should disappear, then select a day, time should appear again

## Regression Notes
1. Potential unintended areas of impact
Reminder scheduling

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Unit testing

PR submission checklist:
